jax/experimental/jax2tf/jax2tf.py

```diff
@@ -52,6 +52,7 @@
 _VALID_SCOPE_REGEX = re.compile("^[A-Za-z0-9.][A-Za-z0-9_.\\/>-]*$")
 _INVALID_SCOPE_CHAR = re.compile("[^A-Za-z0-9_.\\/>-]")
 
+_dtype = np.result_type
```

Comment:
Do we really need this abbreviation? It only saves a few characters.

Reply:
This is what we use in `lax_numpy.py` and I thought it makes the intent clearer at the call-site. Happy to use `np.result_type` directly if you'd prefer.

Reply:
This is minor, but I do prefer `np.result_type`. This is one aspect of Google style guide that I like to use: identifiers from other modules use `module.id` style.